app/views/trainees/employing_schools/_no_results.html.erb

```diff
@@ -13,6 +13,7 @@
 <%= f.hidden_field :employing_school_id, value: :no_results_search_again %>
 
 <%= f.govuk_text_field :no_results_search_again_query,
-                       label: { text: t("components.page_titles.search_schools.search_hint") } %>
+                       label: { text: t("components.page_titles.search_schools.search_hint") },
```

Comment:
I think we can just use https://govuk-form-builder.netlify.app/form-elements/text-field/#input-erb-generating-text-inputs-with-custom-widths to change the widths?

Reply:
@darokel I think that sets the [design system's width class overrides](https://design-system.service.gov.uk/styles/spacing/#width-override-classes), which don't work with the autocomplete - happy to be proved wrong though.

My more general width helpers I made because the default 'design system' way of doing widths is proportional to page / column size - which doesn't make any sense. The field sizes should be related to the content you're taking in, not the page. I've not pushed to use them in our app though (yet‽) and probably use them myself inconsistently in the prototype.

Reply:
See my comment - I don't think this would work with the autocomplete - as that's js generated html and doesn't inherit the styles of the select correctly.

Reply:
Using the design system widths as in [this PR](https://github.com/DFE-Digital/register-trainee-teachers/pull/935/files) works with the autocomplete, however the widths do not accurately match the prototype as in the screenshot below
![image](https://user-images.githubusercontent.com/910019/120489223-61a79d80-c3af-11eb-8053-579e1f7edfcd.png)

Reply:
My bad! But I think the issue would still present if we tried to appropriately set the width of autocompletes enhancing a select - it just happens that this one doesn't start that way.